src/convertToObject.js

```diff
@@ -6,7 +6,26 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const arraySorceString = sourceString
+    .split(';')
+    .map((string) => string.trim().split(':'));
+
+  const filteredSorceString = arraySorceString
+    .filter((innerArr) => innerArr.length === 2)
+    .map((element) => element.map((item) => item.trim()));
+
+  // console.log(filteredSorceString);
+
+  const convertingObject = filteredSorceString.reduce((acc, arrayElement) => {
+    const [key, value] = arrayElement;
+
+    acc[key] = value;
+
+    return acc;
+  }, {});
```

Comment:
The variable 'convertingObject' could be renamed to something like 'styleObject' to better describe what it represents.